README.md

```diff
@@ -79,73 +80,61 @@ One line include following parts:
-
-- **For most developers, it's recommended to start with the [in-depth tutorial for creating a site with Gatsby](https://www.gatsbyjs.org/tutorial/).** It starts with zero assumptions about your level of ability and walks through every step of the process.
-
-- **To dive straight into code samples, head [to the documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
+    ├── package-lock.json
+    ├── README.md
+    ├── scripts/
+    ├── src/
+    └── terraform/
+
+1. **`azure-pipelines.yml`**: Definition of pipeline, used by Azure DevOps. Check 'depended repositories' for [details](#depended-repositories).
+1. **`cli/`**: Internall CLI, more information [here](cli/README.md).
+1. **`dependencies_repos.csv`**: List of depended repositories, more can be read [here](#depended-repositories).
+1. **`Dockerfile`**: Docker for local development.
+1. **`gatsby-config.js`**: This is the main configuration file for a Gatsby site. This is where you can specify information about your site (metadata) like the site title and description, which Gatsby plugins you’d like to include, etc. (Check out the [config docs](https://www.gatsbyjs.org/docs/gatsby-config/) for more detail).
+1. **`gatsby-node.js`**: This file is where Gatsby expects to find any usage of the [Gatsby Node APIs](https://www.gatsbyjs.org/docs/node-apis/) (if any). These allow customization/extension of default Gatsby settings affecting pieces of the site build process.
+1. **`package-lock.json`** (See `package.json` below, first). This is an automatically generated file based on the exact versions of your npm dependencies that were installed for your project. **(You won’t change this file directly).**
+1. **`package.json`**: A manifest file for Node.js projects, which includes things like metadata (the project’s name, author, etc). This manifest is how npm knows which packages to install for your project.
+1. **`README.md`**: A text file containing useful reference information about your project.
+1. **`scripts/`**: All additional scripts. At the moment there is only `get-depended-repos.sh` responsible for pulling depended repos needed by gatsby during build process.
+1. **`src/`**: This directory will contain all of the code related to what you will see on the front-end of your site (what you see in the browser) such as your site header or a page template. `src` is a convention for “source code”.
+1. **`terraform`**: Whole website infrastructure related stuff are served by [Terraform](https://www.terraform.io/).
 
 ## 💫 Deploy
 
-1.  **Build the site for production.**
+1. **Build the site for production.**
 
     Navigate into the site’s directory and run the build command.
 
     ```npm run build```
-
-2.  **Serve the production build locally.**
+1. **Serve the production build locally.**
```

Comment:
**RC** Should be 2 :)

Reply:
Nope, it is markdown https://ibb.co/PxX3n10
https://www.markdownguide.org/basic-syntax/#ordered-lists